Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/IncidentConsumer/IncidentConsumer.cs

```diff
@@ -91,23 +96,23 @@ private string doPUT(string URI, string body, String token, ILogger log)
                 {
                     result = streamReader.ReadToEnd();
                 }
+
                 return result;
             }
             catch (Exception ex)
             {
                 log.LogError("doPUT URI --> " + URI);
                 log.LogError("doPUT body --> " + body);
                 log.LogError("doPUT ex --> " + ex.Message);
+                return null;
             }
-            throw new Exception();
```

Comment:
This line is not needed. The function will either exit on ln 100 or ln 107. This exception re-throwing will never happen.